src/core/Akka.Streams.Tests/ActorMaterializerSpec.cs

```diff
@@ -89,5 +89,12 @@ public void ActorMaterializer_should_report_correctly_if_it_has_been_shut_down_f
             sys.Terminate().Wait();
             m.IsShutdown.Should().BeTrue();
         }
+
+        [Fact]
+        public async Task CanMaterializeStreamsUsingActorSystem()
+        {
+            Func<Task> task = () => Source.Single(1).RunForeach(i => { }, Sys);
+            await task.Should().NotThrowAsync();
```

Comment:
Basic test to test the `ActorSystem` API - under the covers we're actually using the `DefaultMaterializer` in 99.999999% of the Akka.Streams test suite.